Shoko.Server/Databases/SQLite.cs

```diff
@@ -556,6 +556,7 @@ public void CreateDatabase()
             new DatabaseCommand(83, 2, "ALTER TABLE VideoLocal_User ADD WatchedCount INT NOT NULL DEFAULT 0;"),
             new DatabaseCommand(83, 3, "ALTER TABLE VideoLocal_User ADD LastUpdated timestamp NOT NULL DEFAULT '2000-01-01 00:00:00';"),
             new DatabaseCommand(83, 4, "UPDATE VideoLocal_User SET WatchedCount = 1, LastUpdated = WatchedDate WHERE WatchedDate IS NOT NULL;"),
+            new DatabaseCommand(84, 1, "ALTER TABLE AnimeSeries_User ADD LastUpdated timestamp NOT NULL DEFAULT '2000-01-01 00:00:00';"),
```

Comment:
why not `DEFAULT CURRENT_TIMESTAMP` ?

Reply:
Currently the version of SQLite in use doesn't support it... that's why.

Reply:
Forgot to try and update the dependencies

Reply:
Will try later today or tomorrow